README.md

```diff
@@ -11,11 +21,7 @@
   - [Maintainers](#maintainers)
   - [Preparing Releases](#preparing-releases)
 
-NHS.UK Frontend ported to React
-
-[![GitHub Actions CI Status](https://github.com/NHSDigital/nhsuk-react-components/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/NHSDigital/nhsuk-react-components/actions?query=workflow%3A%22CI+Build%22+branch%3Amain) [![Bundle Size](https://img.shields.io/bundlephobia/minzip/nhsuk-react-components.svg)](https://bundlephobia.com/result?p=nhsuk-react-components)
-
-## Coming from 0.x?
+## Coming from 0.x
```

Comment:
Could consolidate all these headings and links into a single heading with a list, to avoid all the "this doc" links?

Something like:

```
## Upgrade guide

* [Upgrading to 1.0](/docs/upgrade-to-1.0.md)
* [Upgrading to 2.0](/docs/upgrade-to-2.0.md)
* [Upgrading to 3.0](/docs/upgrade-to-3.0.md)
* [Upgrading to 4.0](/docs/upgrade-to-4.0.md)
```